src/follower.py

```diff
@@ -37,17 +38,16 @@ def __init__(self):
         self._camera = Camera()
         self.reset_camera()
 
-        self._tag = TagRecognition()
+        self._tag = TagRecognition(marker_length=0.025)
         self._speed = 0
 
         self._distance = 0
         self._turn_angle = 0
         self._decision = 0
-
+        self._yaw = 0
 
     """
     Drives the vehicle forward and avoids collisions with recognized objects.
-
```

Comment:
Don't delete the blank line between the first line of the comment and the rest of the comment. It's there for a reason.  Get rid of this change here and everywhere else you did it.